pyo3-build-config/src/impl_.rs

```diff
@@ -1560,6 +1675,9 @@ mod tests {
             parsed_config,
             InterpreterConfig {
                 abi3: false,
+                abi_flags: Some("m".to_string()),
+                abi_tag: Some("37m".to_string()),
+                ext_suffix: Some(".cpython-37m-x86_64-linux-gnu.so".to_string()),
```

Comment:
I think this `assert_eq` needs to change to be successful even when these fields don't match exactly, as per the note above:
>  // A best effort attempt to get test coverage for the sysconfigdata parsing.
// Might not complete successfully depending on host installation; that's ok as long as
// CI demonstrates this path is covered!

Reply:
Oh I see what's happening - this test is conditional for x86_64 linux but I don't know how to make it conditional on Python version in CI.